wrangler-core/src/test/java/io/cdap/wrangler/TestingPipelineContext.java

```diff
@@ -81,6 +83,16 @@ public void pipelineCount(String s, int i) {
       public void pipelineGauge(String s, long l) {
 
       }
+
+      @Override
+      public Metrics child(Map<String, String> tags) {
+        return this;
+      }
+
+      @Override
+      public Map<String, String> getTags() {
+        return new HashMap<>();
```

Comment:
`return Collections.emptyMap();`

to return immutable map.

Reply:
Fixed, thanks for review!